etc/helm/pachyderm/templates/pachd/deployment.yaml

```diff
@@ -150,6 +150,10 @@ spec:
         - name: SSL_CERT_DIR
           value:  /pachd-tls-cert
         {{- end }}
+        {{- if .Values.pachd.tls.enabled }}
```

Comment:
shouldn't this be if global.customcacerts?

Reply:
I think either this or `{{- if and .Values.pachd.tls.enabled .Values.global.customCaCerts }}` as we have elsewhere should work. Right now the only use-case to pass the secret name to pachd is for the `customCaCerts` case, but this value is more general. I can change it to be consistent

Reply:
Lgtm. Testing locally now. I'll package up my setup so we can make a permanent test